### PR TITLE
Enable line length testing in PEP8 test with max length of 85

### PR DESCRIPTION
**Pull request by yongzx**
In https://github.com/keras-team/keras/projects/1, one improvement is to "Enable line length testing in PEP8 test (set to a reasonable value like 85)."

### CODE REVIEW

**Comment by yongzx** [comment]
The failing check is due to the violation of the newly added pep8 test of line lengths. Should I modify those files that result in the fail tests?

**Comment by Dref360** [comment]
Please do

**Comment by yongzx** [comment]
I have reformatted all the files.

File: examples/deep_dream.py [hunk 1]
```diff
@@ -83,7 +83,8 @@ def deprocess_image(x):
 loss = K.variable(0.)
 for layer_name in settings['features']:
     # Add the L2 norm of the features of a layer to the loss.
-    assert layer_name in layer_dict.keys(), 'Layer ' + layer_name + ' not found in model.'
+    assert layer_name in layer_dict.keys(), 'Layer ' + layer_name + \
```
Comment (Dref360) [inline examples/deep_dream.py]: same

File: examples/conv_filter_visualization.py [hunk 2]
```diff
@@ -129,8 +129,10 @@ def normalize(x):
 for i in range(n):
     for j in range(n):
         img, loss = kept_filters[i * n + j]
-        stitched_filters[(img_width + margin) * i: (img_width + margin) * i + img_width,
-                         (img_height + margin) * j: (img_height + margin) * j + img_height, :] = img
+        stitched_filters[(img_width + margin) * i:
+                         (img_width + margin) * i + img_width,
```
Comment (Dref360) [inline examples/conv_filter_visualization.py]: We kind lose the meaning of this line

File: examples/image_ocr.py [hunk 3]
```diff
@@ -88,16 +88,21 @@ def paint_text(text, w, h, rotate=False, ud=False, multi_fonts=False):
         context.paint()
         # this font list works in CentOS 7
         if multi_fonts:
-            fonts = ['Century Schoolbook', 'Courier', 'STIX', 'URW Chancery L', 'FreeMono']
-            context.select_font_face(np.random.choice(fonts), cairo.FONT_SLANT_NORMAL,
-                                     np.random.choice([cairo.FONT_WEIGHT_BOLD, cairo.FONT_WEIGHT_NORMAL]))
+            fonts = ['Century Schoolbook', 'Courier', 'STIX', 'URW Chancery L',
+                     'FreeMono']
+            context.select_font_face(np.random.choice(fonts),
+                                     cairo.FONT_SLANT_NORMAL,
```
Comment (Dref360) [inline examples/image_ocr.py]: As a rule of thumb, if you put an arg on another line do so for each argument.
x = [super_long_a, super_long_b,super_long_c]

x = [super_long_a,
     super_long_b,
     super_long_c]

File: examples/image_ocr.py [hunk 4]
```diff
@@ -217,7 +222,8 @@ def build_word_list(self, num_words, max_string_len=None, mono_fraction=0.5):
                 if len(tmp_string_list) == int(self.num_words * mono_fraction):
                     break
                 word = line.rstrip()
-                if max_string_len == -1 or max_string_len is None or len(word) <= max_string_len:
+                if max_string_len == -1 or max_string_len is None or \
```
Comment (Dref360) [inline examples/image_ocr.py]: same

File: examples/image_ocr.py [hunk 5]
```diff
@@ -274,9 +282,11 @@ def get_batch(self, index, size, train):
                 source_str.append('')
             else:
                 if K.image_data_format() == 'channels_first':
-                    X_data[i, 0, 0:self.img_w, :] = self.paint_func(self.X_text[index + i])[0, :, :].T
+                    X_data[i, 0, 0:self.img_w, :] = self.paint_func(
```
Comment (Dref360) [inline examples/image_ocr.py]: indentation make the line unclear

File: examples/image_ocr.py [hunk 6]
```diff
@@ -447,50 +470,63 @@ def train(run_name, start_epoch, stop_epoch, img_w):
                    name='conv2')(inner)
     inner = MaxPooling2D(pool_size=(pool_size, pool_size), name='max2')(inner)
 
-    conv_to_rnn_dims = (img_w // (pool_size ** 2), (img_h // (pool_size ** 2)) * conv_filters)
+    conv_to_rnn_dims = (img_w // (pool_size ** 2), (img_h // (pool_size ** 2)) *
```
Comment (Dref360) [inline examples/image_ocr.py]: indentation

**Comment by yongzx** [comment]
Thanks for checking my PR. I will fix the mistakes you have pointed out first before you continue checking (so as not to waste your time.) However, I have a question. I put down `#noqa` for some documentations and parameters involving url_path because I don't want to split them. Do you want me to split them up if the url is too long?

**Comment by Dref360** [comment]
for urls it's fine I think.

**Comment by taehoonlee** [comment]
This PR looks awesome! but it is so huge as @Dref360 said :)
@yongzx, [here](https://github.com/keras-team/keras-applications/blob/master/keras_applications/resnet50.py#L31) is an example for URLs.

**Comment by yongzx** [comment]
But should I break down the url if it is in the docstring?

**Comment by taehoonlee** [comment]
@yongzx, In most cases, you can simply put a single URL on a single line. Because pep8 will skip line-length checking in case of a single long word. For example,

```python
'''
SUPERLONGWORD
SUPERLONGWORD SUPERLONGWORD
http://SUPERLONGURL
SUPERLONGWORD http://SUPERLONGURL
'''
a = 'SUPERLONGWORD'
b = 'SUPERLONGURL'
```

```
test.py:3:80: E501 line too long
test.py:5:80: E501 line too long
test.py:7:80: E501 line too long
test.py:8:80: E501 line too long
```

It would be better not to break, but it depends on the situation I think.

**Comment by yongzx** [comment]
I learn something new. However, take this line from the docstring in examples/mnist_net2net.py. It is no longer two words, and it seems like if I put in a new line into the docstring it may break the transformation of it into a markdown file. What should I do?

- [A Hierarchical Neural Autoencoder for Paragraphs and Documents](https://arxiv.org/abs/1506.01057)

**Comment by taehoonlee** [comment]
@yongzx, A new line between `]` and `(` does not break a markdown link.

**Comment by yongzx** [comment]
Hi, I am new to PR, and I am still working on the files. I know there are merge conflicts now due to new PRs by other people. Should I still continue working on my local files (aka the files that I have been using to do the previous PRs)? Thank you.

**Comment by taehoonlee** [comment]
@yongzx, Yes, you should resolve the conflicts.

**Comment by gabrieldemarmiesse** [comment]
Maybe a solution to avoid those conflicts might be to do smaller PRs? See #10698

**Comment by yongzx** [comment]
Thanks @gabrieldemarmiesse! I will proceed to smaller PR.

**Pull request closed without merging**